app/src/main/res/drawable/start_button_transparent_background.xml

```diff
@@ -6,6 +6,9 @@
         <shape>
           <corners android:radius="4dp" />
           <solid android:color="@color/oppia_disabled_button_background" />
+          <stroke
+            android:width="1dp"
+            android:color="@color/component_color_profile_rename_activity_inactive_save_button_border_color" />
```

Comment:
Rename to `component_color_start_button_transparent_background_sroke_color `

Reply:
updated

Reply:
done